File: Go/FibonacciNumbers.go
package main

import "fmt"

func fibonacciSum(n int) int{
	//Handle cases for n = 1 and n=2
	if n==1 {
		return 1;
	}
	if n==2 {
		return 2;
	}

	//initiate first two values
	a := 1
	b := 1
	//inititate suma and adjust n accordingly
	sum := 2
	n-=2

	//get sum for n values in fibonacci series
	for n!=0{
		a,b = b,a+b
		sum+=b
		n--
	}
	return sum
}

func main(){
//Read value of n
 fmt.Println("Enter value of 'n'(integer):")
 var n int
 _,err := fmt.Scanf("%d",&n);
 if err!=nil{
 	fmt.Println("Invalid input")
 	return
 }
 fmt.Printf("Sum till nth term:%d",fibonacciSum(n))
}

/*
I/O =>
Enter value of 'n'(integer):
4
Sum till nth term:7

Time complexity: O(n)
Space complexity: O(1)
*/

File: PHP/fibonacci_numbers.php
<?php
/*
The Fibonacci numbers are the numbers in the following integer sequence.
0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, ……..

Given the value of n, find the sum of the series.
*/

$n= intval(readline("Enter the value of n:"));
$fn1 = 1;
$fn2 = 0;
$sum=0;
for($i = 0; $i < $n+1; $i++){
  $sum=$fn1+$fn2;
  $t=$fn2 + $fn1;
  $fn2 =$fn1;
  $fn1 = $t;
}
echo strval($sum-1)."\n";
/*
Example:
Input: 9
Output: 88
Time complexity: O(n)
Space complexiity: O(1)
*/

?>